package.json

```diff
@@ -25,8 +25,8 @@
     "eslint-plugin-prettier": "^3.4.0",
     "prettier": "^2.3.0",
     "prettier-eslint": "^12.0.0",
-    "typedoc": "^0.24.7",
```

Comment:
@rugoncalves Probably this will also happen on other assets

Reply:
possibly... I'll check.